src/components/callToAction/CallToAction.styles.ts

```diff
@@ -15,10 +15,10 @@ export const CallToActionStyled = styled.div<{
   width: fit-content;
   box-shadow: 4px 4px 0 0 #000;
   text-align: center;
-  font-size: 16px;
-  font-weight: 500;
-  font-family: var(--primary-font);
-  letter-spacing: -0.24px;
+  font-size: ${({ theme }) => theme.fontSizes.largeParagraph}px;
+  font-weight: ${({ theme }) => theme.fontWeights.medium};
+  font-family: ${({ theme }) => theme.fonts.primaryFont};
```

Comment:
font-family is best set somewhere in global styles, it's painful to declare it in each component

Reply:
it was added here cause agent styles were overriding font family set to body in globalStyles.ts. Now, I've added to globalStyles.ts this:
```
button {
  font-family: ${({ theme: { fonts } }) => fonts.primaryFont};
}
```
Will it be a better solution?